exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/trace_exporter/__init__.py

```diff
@@ -290,6 +290,9 @@ def _translate_data(
     def export(self, spans: Sequence[ReadableSpan]) -> SpanExportResult:
         return self._export(spans)
 
+    def shutdown(self) -> None:
+        OTLPExporterMixin.shutdown(self)
```

Comment:
@girishc13 Can you please help me understand why you are calling `shutdown` on mixin class directly instead of using `super()`?

Reply:
I don't remember the exact details but the `shutdown` method is implemented by both the `OTLPExporterMixin` and the `SpanExporter` interfaces. The exporter.shutdown is handled by different logic and this pr was targeting the backend client that needs to be shutdown. You need to trace the calls for shutdown from the usage of the `OTLPSpanExporter`.